### PR TITLE
fix: remove helm 2 support

### PR DESCRIPTION
**Pull request by innobead**
ref: longhorn/longhorn#6318

### CODE REVIEW

File: content/docs/1.6.0/advanced-resources/deploy/customizing-default-settings.md [hunk 1]
```diff
@@ -96,23 +96,18 @@ From the project view in Rancher, go to **Apps && Marketplace > Longhorn > Insta
 
 ### Using Helm
 
+> **NOTE:**
+> Helm 2 is not supported, [ref](https://helm.sh/blog/helm-2-becomes-unsupported/).
+> Please use Helm 3 for Longhorn installation and upgrade.
```
Comment (jillian-maroket) [inline content/docs/1.6.0/advanced-resources/deploy/customizing-default-settings.md:101]:
```
> Use Helm 3 when installing and upgrading Longhorn. Helm 2 is [no longer supported](https://helm.sh/blog/helm-2-becomes-unsupported/).
```

File: content/docs/1.6.0/deploy/install/install-with-helm.md [hunk 2]
```diff
@@ -8,9 +8,7 @@ In this section, you will learn how to install Longhorn with Helm.
 ### Prerequisites
 
 - Each node in the Kubernetes cluster where Longhorn will be installed must fulfill [these requirements.](../#installation-requirements)
```
Comment (jillian-maroket) [inline content/docs/1.6.0/deploy/install/install-with-helm.md:10]:
```
- Kubernetes cluster: Ensure that each node fulfills the [installation requirements](../#installation-requirements).
```

File: content/docs/1.6.0/deploy/install/install-with-helm.md [hunk 3]
```diff
@@ -8,9 +8,7 @@ In this section, you will learn how to install Longhorn with Helm.
...
-- Helm v2.0+ must be installed on your workstation.
-  - 1. Refer to the official documentation for help installing Helm.
-  - 2. If you're using a Helm version prior to version 3.0, you need to [install Tiller in your Kubernetes cluster with role-based access control (RBAC)](https://v2.helm.sh/docs/using_helm/#tiller-namespaces-and-rbac).
+- Helm v3 must be installed on your workstation.
```
Comment (jillian-maroket) [inline content/docs/1.6.0/deploy/install/install-with-helm.md:11]:
```
- Your workstation: Install [Helm](https://helm.sh/docs/) v3.0 or later.
```

File: content/docs/1.6.0/deploy/install/install-with-helm.md (hunk 2 above)
Comment (jillian-maroket) [reply content/docs/1.6.0/deploy/install/install-with-helm.md:10]: This is the exact wording in the "Install with Flux" and "Install with Fleet" files. I suggest we use the same here.

File: content/docs/1.6.0/deploy/install/install-with-helm.md (hunk 3 above)
Comment (jillian-maroket) [reply content/docs/1.6.0/deploy/install/install-with-helm.md:11]: This is the exact wording in the "Install with Flux" and "Install with Fleet" files. I suggest we use the same here.

**Review by jillian-maroket — COMMENTED** [review]
I don't know why the PR was merged before the review.

**Comment by innobead** [comment]
> I don't know why the PR was merged before the review.

This was a minor fix, so I merged it directly. Let's adopt your feedback here. I will have another PR.

**Pull request merged**